autogpt_platform/backend/backend/data/db.py

```diff
@@ -1,4 +1,3 @@
-import asyncio
```

Comment:
No change here, just connection code tidying.
I decoupled the retry logic and moved it to retry.py.